src/main/java/task/TaskManager.java

```diff
@@ -0,0 +1,99 @@
+package task;
+
+import java.util.ArrayList;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.Scanner;
+
+public class TaskManager {
+    static protected ArrayList<Task> taskList;
+    static protected Task task;
+
+    public TaskManager() {
+        taskList = new ArrayList<Task>();
+    }
+
+    public static int getTaskCount() {
+        return taskList.size();
+    }
+
+    public Task getTask(int index) {
+        return taskList.get(index);
+    }
+
+    public void addToList(Task task) {
+        taskList.add(task);
+
+        System.out.println("Got it. I've added this task:");
+    }
+
+    public void markAsCompleted(int index) {
+        Task task = taskList.get(index);
+        task.markAsCompleted();
```

Comment:
referring to my comment at task.java, you can shift the print statements here, I think it would be a more appropriate level.